artman/cli/converter.py

```diff
@@ -0,0 +1,297 @@
+            legacy_common.get('service_yaml')[0])
+    if legacy_common.get('proto_deps'):
+        result.common.proto_deps.extend(
+            _compute_deps(legacy_common.get('proto_deps')))
+    if legacy_common.get('proto_test_deps'):
+        result.common.test_proto_deps.extend(
+            _compute_deps(legacy_common.get('proto_test_deps')))
+    if legacy_common.get('src_proto_path'):
+        result.common.src_proto_paths.extend(
+            _compute_src_proto_paths(legacy_common.get('src_proto_path')))
+
+    result.artifacts.extend(_compute_artifacts(legacy_config, legacy_common))
+    return result
+
+
+def _compute_artifacts(legacy_config, legacy_common):
+    result = []
+    LANGS = ['java', 'python', 'php', 'ruby', 'go', 'csharp', 'nodejs']
+
+    # Compute artifacts section
+    if legacy_common.get('git_repos'):
+        common_git_repos = legacy_common.get('git_repos')
+    else:
+        common_git_repos = []
+
+    for lang in LANGS:
+        if lang not in legacy_config:
+            continue
+        legacy_artifact_config = legacy_config[lang]
+        if legacy_artifact_config:
+            artifact = Artifact()
+            # Name the artifact as `{lang}_gapic`.
+            artifact.name = '%s_gapic' % lang
+            artifact.language = Artifact.Language.Value(lang.upper())
+
+            if 'release_level' in legacy_artifact_config:
+                artifact.release_level = Artifact.ReleaseLevel.Value(
+                    legacy_artifact_config.get('release_level').upper())
+
+            artifact.type = _compute_artifact_type(legacy_common)
+
+            # Compute package version.
+            if 'generated_package_version' in legacy_artifact_config:
+                legacy_package_version = legacy_artifact_config.get(
+                    'generated_package_version')
+                if 'lower' in legacy_package_version:
+                    artifact.package_version.grpc_dep_lower_bound = (
+                        legacy_package_version.get('lower'))
+                if 'upper' in legacy_package_version:
+                    artifact.package_version.grpc_dep_upper_bound = (
+                        legacy_package_version.get('upper'))
+
+            # Compute publishing targets
+            if 'git_repos' in legacy_artifact_config:
+                artifact.publish_targets.extend(_compute_publish_targets(
+                    legacy_artifact_config.get('git_repos'), common_git_repos))
+            result.append(artifact)
+    return result
+
+
+def _compute_artifact_type(legacy_common):
+    if legacy_common.get('packaging', '') == 'google-cloud':
+        return Artifact.GAPIC_ONLY
+    elif legacy_common.get('package_type', '') == 'grpc_common':
+        return Artifact.GRPC_COMMON
+
+    return Artifact.GAPIC
+
+
+def _compute_publish_targets(git_repos, common_git_repos):
+    result = []
+    for k, v in git_repos.items():
+        location = (common_git_repos[k]['location']
```

Comment:
Luke, my comment comes from the Google style guide: https://google.github.io/styleguide/pyguide.html?showone=Conditional_Expressions#Conditional_Expressions